Ports/ClassiCube/patches/0003-HACK-Disable-interrupt-hooks-on-Serenity.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: cflip <cflip@cflip.net>
+Date: Sat, 15 Oct 2022 15:29:44 -0600
+Subject: [PATCH] HACK: Disable interrupt hooks on Serenity
+
+This allows crashes to be handled by Serenity's crash handler, making
+debugging much easier.
+---
+ Program.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/Program.c b/Program.c
+index 3b210522036a51a8f4a9fece8b5b2e20be79a7aa..3bc730ce8f43ae88b313048e39dfb7ab8c3ffc06 100644
+--- a/Program.c
++++ b/Program.c
+@@ -55,7 +55,9 @@ CC_NOINLINE static void WarnMissingArgs(int argsCount, const cc_string* args) {
+ static void SetupProgram(int argc, char** argv) {
+ 	static char ipBuffer[STRING_SIZE];
+ 	cc_result res;
++#ifndef CC_BUILD_SERENITY
+ 	Logger_Hook();
++#endif

```

Comment:
Instead of `#ifndef`ing it out, you can just patch it to remove the `Logger_Hook();` line. The patches are only applied for SerenityOS anyway.